qcodes/instrument_drivers/QuantumDesign/DynaCoolPPMS/DynaCool.py

```diff
@@ -0,0 +1,79 @@
+from qcodes.instrument.visa import VisaInstrument
```

Comment:
could you add a general description of the architecture? that you have a server, driver, visa-like commands, blabla

Reply:
I think this is in the notebook, Is that enough?

Reply:
yes, that should be enough.